src/hist_file.rs

```diff
@@ -46,6 +47,11 @@ pub fn parse_contents(contents: String, args: &Args) -> HashMap<String, usize> {
         });
         only_prefix.push_str("\n");
     }
+
+    let reg = Regex::new(r": \d\d\d\d\d\d\d\d\d\d:\d;").unwrap();
```

Comment:
Could it be possible to let users provide this regex as an arguement? That way, we could cover similar edge cases in other shells as well

Reply:
Alright I'll work on adding that. It might take a while though, as I'm pretty new to rust.

Reply:
That's alright. I appreciate any help